tests/test_multiset.py

```diff
@@ -895,13 +895,18 @@ def test_dict_methods(MultisetCls):
 
 def test_mutating_dict_methods():
     ms = Multiset('aab')
+    initial_size = len(ms)
 
     assert ms.pop('a', 5) == 2
+    assert len(ms) == initial_size - 2
```

Comment:
I think it would be clearer to avoid computing these values inside a test. In the first step it is still understandable, but after that it doesn't really explain well what is expected.

So I would suggest to either compare it with a constant or do something like this to make clear what the expected state of the multiset is:
```suggestion
    assert len(ms) == len('b')
```